src/Resources/Resources.Test/Models.ResourceGroups/ResourceClientTests.cs

```diff
@@ -1153,108 +1152,6 @@ public void GetsAllResourceGroupsWithDetails()
             Assert.Equal(resourceGroup4.Name, actual[3].ResourceGroupName);
         }
 
-        [Fact]
```

Comment:
Hi why these two test cases need to be deleted?

Reply:
These tests were checking that the client side filtering was working on the data retrieved from server.

Now there is no client side filtering. The data is filtered on the server, and we have tests on server side to check appropriate filtering.